docs/permission/plugin-authors/setup-for-the-tutorial.md

```diff
@@ -0,0 +1,126 @@
+---
+id: setup-for-the-tutorial
+title: Permission framework for plugin authors
+description: How to get started with the permission framework as a plugin author
+---
+
+The following tutorial is designed to help plugin authors add support for permissions to their plugins. We'll add support for permissions to example `todo-list` and `todo-list-backend` plugins, but the process should be similar for other plugins!
+
+The rest of this page is focused on adding the `todo-list` and `todo-list-backend` plugins to your Backstage instance. If you want to add support for permissions to your own plugin instead, feel free to skip to the [next section](authorize-the-create-endpoint.md).
+
+## Setup for the Tutorial
+
+We will use a "Todo list" feature, composed of the `todo-list` and `todo-list-backend` plugins.
+
+The source code is available here:
+
+- [todo-list](https://github.com/backstage/backstage/blob/master/contrib/plugins/todo-list)
+- [todo-list-backend](https://github.com/backstage/backstage/blob/master/contrib/plugins/todo-list-backend)
+
+1.  Copy-paste the two folders into the plugins folder of your backstage application repository or run the following script from the root of your backstage application:
+
+    ```bash
+      $ node -e "const{UrlReaders}=require('@backstage/backend-common');\
```

Comment:
Worth considering something a little briefer in bash maybe? Not cross-platform obviously, but perhaps worth it for the brevity:

```bash
curl https://codeload.github.com/backstage/backstage/zip/refs/heads/master | tar --strip-components=1 -xv backstage-master/contrib/plugins/todo-list backstage-master/contrib/plugins/todo-list-backend
```

Reply:
The file is about 100mb, which doesn't seem prohibitive in an npm world 😂